File: statsig-ffi/bindings/java/src/main/java/com/statsig/DynamicConfig.java
package com.statsig;

import com.google.gson.JsonElement;

import java.util.Map;

public class DynamicConfig extends Experiment {
    DynamicConfig(String name, Map<String, JsonElement> value, String ruleID, EvaluationDetails evaluationDetails,
            String idType) {
        super(name, value, ruleID, null, evaluationDetails, idType);
    }
}

File: statsig-dotnet/dotnet-statsig/DynamicConfig.cs
using System.Collections.Generic;
using Newtonsoft.Json.Linq;

namespace Statsig
{
    public class DynamicConfig : Experiment
    {
        
        internal DynamicConfig(string name, string ruleId, IReadOnlyDictionary<string, JToken> value, EvaluationDetails details)
            : base(name, ruleId, value, null, details)  // Passing null for GroupName
        {
        }
    }
}